enclave/sgx/revocationinfo.c

```diff
@@ -166,3 +166,178 @@ void oe_free_get_revocation_info_args(oe_get_revocation_info_args_t* args)
         free(args->buffer);
     }
 }
+
+/**
+ * Call into host to fetch collateral information.
+ */
+oe_result_t oe_get_quote_verification_collateral(
```

Comment:
add _sgx_ into function name, for consistency with line 174

Reply:
Thanks, function name changed.